### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by aubinleyoung**
### In this milestone, I was able to implement the following requirements:

- Create a new directory called logic inside the project
- Copy the files calculate.js and operate.js into the logic/ directory
- Install big.js using NPM
- Import the files in my Calculator.js component
- Implement click event handlers using the math logic from calculate.js and operate.js in my React component
- Test the app in the browser. The Calculator should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,27 +1,70 @@
...
+      <button type="button" className="six btn" onClick={operateEvents}>9</button>
+      <button type="button" className="seven btn" onClick={operateEvents}>x</button>
+      <button type="button" className="eigth btn" onClick={operateEvents}>4</button>
+      <button type="button" className="nine btn" onClick={operateEvents}>5</button>
+      <button type="button" className="ten btn" onClick={operateEvents}>6</button>
+      <button type="button" className="eleven btn" onClick={operateEvents}>-</button>
+      <button type="button" className="twelve btn" onClick={operateEvents}>1</button>
+      <button type="button" className="thirteen btn" onClick={operateEvents}>2</button>
+      <button type="button" className="fourteen btn" onClick={operateEvents}>3</button>
+      <button type="button" className="fifteen btn" onClick={operateEvents}>+</button>
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] You have two plus signs (lines 23 and 35). Kindly note that the first one should be a division sign ("&div;", not "/"). Ensure you add the correct sign and that the operation works as required

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @aubinleyoung,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -33,8 +33,8 @@
```
Comment (HAMMAS-SALEEM) [inline README.md:33]: image

**[Optional]**

- [ ] Kindly remove the unnecessary sections from your application such as shown in the image above to make your readme file clean, professional and descriptive :blush:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @aubinleyoung 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
:heavy_check_mark: Following Gitflow

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**